### ISSUE TITLE
El Juguete Perdido – Combat: Visual issues when using aspect ratios other than 16:9

### ISSUE THREAD

**Issue by russell-cj** [endlessm/threadbare#1552]
In the scene `2_combat/combat.tscn`, some backgrounds or decorative elements appear outside the visible playable area when using aspect ratios other than 16:9.

Gray areas are visible within the playable area of the map in `2_combat/combat.tscn`, in an environment that should be covered with decorative elements, such as stones, to maintain the visual consistency of the level.

### Quick Steps
1. Open the scene in **Godot**.  
2. Enable **Debug Aspect Ratio**.  
3. Change the window aspect ratio (e.g., 21:9 or 4:3) and check for elements out of place.

> [!NOTE]  
> - Sub-issue of main issue **El Juguete Perdido: Visual issues when using aspect ratios other than 16:9 #1549**  
> - Scene path: `res://scenes/quests/story_quests/el_juguete_perdido/2_combat/combat.tscn`
